### ISSUE TITLE
5.13.0 Breaks html plugin

### ISSUE THREAD

**Issue by plinss** [eslint/eslint#11346]
**Tell us about your environment**

* **ESLint Version:**
5.13.0

* **Node Version:**
Docker node:latest (running in ci environment)

* **npm Version:**
Docker node:latest

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
	"extends": ["eslint:recommended", "google"],
	"parser": "babel-eslint",
	"parserOptions": {
		"ecmaVersion": 2018,
		"sourceType": "module",
	    "allowImportExportEverywhere": true
    },
	"env": {
		"browser": true
	},
	"plugins": [
		"html",
		"json",
		"async-await"
	],
	"rules": {
		"brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
		"new-cap": ["error", { "capIsNewExceptions": ["Polymer"] }],
		"require-jsdoc": "off",
		"max-len": ["warn", {
			"code": 160,
			"ignorePattern": "<g id=\"icon",
			"ignoreComments": true,
			"ignoreTrailingComments": true,
			"ignoreTemplateLiterals": true
		}],
		"no-extend-native": "off",
		"no-tabs": "off",
		"indent": ["error", "tab", { "SwitchCase": 1, "ignoreComments": true }],
		"no-mixed-spaces-and-tabs": ["error", "smart-tabs"],
		"object-curly-spacing": ["error", "always"],
		"block-spacing": ["error", "always"],
		"space-in-parens": ["error", "never"],
		"arrow-parens": ["error", "as-needed", { "requireForBlockBody": true }],
		"quotes": ["error", "single", { "avoidEscape": true }],
		"no-multi-spaces": ["error", { "exceptions": { "Property": true, "VariableDeclarator": true, "ImportDeclaration": true } }],
		"space-infix-ops": ["error", {"int32Hint": false}],
		"space-unary-ops": ["error", {
			"words": true,
			"nonwords": false,
			"overrides": {
				"!": true,
				"!!": true
			}
	    }],
	    "no-unused-vars": ["error", { "vars": "all", "args": "none", "ignoreRestSiblings": false, "varsIgnorePattern": "^_$" }],
		"prefer-const": "off",
		"camelcase": ["error", {
			"allow": ["content_type", "include_docs", "skip_setup", "update_seq", "inclusive_end",
			          "accountStatus_*", "processingStatus_*", "payoutStatus_*", "sellerStatus_*"]
		}],
		"require-await": "error",
		"require-yield": "error"
	},
	"globals": {
		"Polymer": true,
		"Set": true,
		"Map": true,
		"Uint8Array": true,
		"DataView": true,
		"Promise": true,
		"Symbol": true
	}
}

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**
removing the html plugin resolves the problem.

```js
n/a
```

```bash
npm install -g eslint eslint-plugin-html eslint-plugin-json eslint-plugin-async-await eslint-config-google babel-eslint
eslint . --ext .js --ext .html --ext .json
```

**What did you expect to happen?**
eslint would run

**What actually happened? Please include the actual, raw output from ESLint.**
TypeError: Cannot read property 'define' of undefined
    at verifyCodePart (/usr/local/lib/node_modules/eslint-plugin-html/src/index.js:167:18)
    at Linter.verify (/usr/local/lib/node_modules/eslint-plugin-html/src/index.js:198:9)
    at Linter.verifyAndFix (/usr/local/lib/node_modules/eslint/lib/linter.js:1032:29)
    at processText (/usr/local/lib/node_modules/eslint/lib/cli-engine.js:197:32)
    at processFile (/usr/local/lib/node_modules/eslint/lib/cli-engine.js:241:12)
    at fileList.map.fileInfo (/usr/local/lib/node_modules/eslint/lib/cli-engine.js:613:40)
    at Array.map (<anonymous>)
    at CLIEngine.executeOnFiles (/usr/local/lib/node_modules/eslint/lib/cli-engine.js:585:34)
    at Object.execute (/usr/local/lib/node_modules/eslint/lib/cli.js:197:111)
    at Object.<anonymous> (/usr/local/lib/node_modules/eslint/bin/eslint.js:78:28)

**Are you willing to submit a pull request to fix this bug?**
Not at this time

**Comment by plinss** [comment]
Thanks for the quick reply, I found an existing issue: https://github.com/BenoitZugmeyer/eslint-plugin-html/issues/113#issuecomment-459883425
